pkgs/applications/graphics/awesomebump/default.nix

```diff
@@ -1,26 +1,37 @@
-{ lib, stdenv, fetchurl, qtbase, qmake, makeWrapper }:
+{ lib, stdenv, fetchurl, qtbase, qmake, makeWrapper, qtscript, gcc, flex, bison, qtdeclarative, unzip }:
 
 stdenv.mkDerivation {
-  name = "awesomebump-4.0";
+  name = "awesomebump-5.1";
 
   src = fetchurl {
-    url = https://github.com/kmkolasinski/AwesomeBump/archive/Linuxv4.0.tar.gz;
-    sha256 = "1rp4m4y2ld49hibzwqwy214cbiin80i882d9l0y1znknkdcclxf2";
+    url = https://github.com/kmkolasinski/AwesomeBump/archive/Winx32v5.1.tar.gz;
+    sha256 = "04s0jj9gfw1rfr82ga2vw6x1jy00ca9p9s3hh31q3k5h6vg5ailn";
   };
 
-  setSourceRoot = "sourceRoot=$(echo */Sources)";
-
-  nativeBuildInputs = [ makeWrapper qmake ];
-  buildInputs = [ qtbase ];
-
-  enableParallelBuilding = true;
+  buildInputs = [ qtbase qtscript qtdeclarative flex bison unzip ];
+
+  nativeBuildInputs = [ qmake makeWrapper ];
+
+  buildPhase = ''
+    cd Sources/utils/QtnProperty
+    unzip "${fetchurl { url = "https://github.com/kmkolasinski/QtnProperty/archive/master.zip"; sha256 = "1hd2bqxhv2w26b5xs5hwxc0s8q8z0bvlkz9zpnr814v8kwkqwp21";  } }"
```

Comment:
Surely `master` can change?

Reply:
Yes, good catch. I've hard-coded the git revision now.